pkg/reconciler/pipelinerun/cancel.go

```diff
@@ -187,7 +224,11 @@ func getChildObjectsFromPRStatusForTaskNames(ctx context.Context, prs v1beta1.Pi
 		}
 		for runName, runStatus := range prs.Runs {
 			if taskNames.Len() == 0 || taskNames.Has(runStatus.PipelineTaskName) {
-				runNames = append(runNames, runName)
+				if cfg.FeatureFlags.CustomTaskVersion == "v1alpha1" {
```

Comment:
can we use the constant for "v1alpha1" instead?

Reply:
ha, missed that, fixing.